### DISCUSSION TITLE
Lock Player - What does it do

### DISCUSSION THREAD

**Discussion by truthsword** [advplyr/audiobookshelf-app#826] [General]
While I admit I'm not the best informed book on the shelf, after toggling "lock player" on/off I'm missing what this feature exactly does and/or what purpose it serves. 

I'm using the Android and running from docker. TIA.

**Comment by nichwall** [comment]
Locking the player gets rid of the skip forward/back buttons and prevents seeking on the progress bar, so you can only play/pause and not accidentally seek.

This does not disable seeking in the notification, that's still an outstanding feature request.

**Reply by truthsword** [reply]
Thank you for the clarification. 

Sorry for the delayed response. Apparently my email reply did not get through.
